concordium-contracts-common-derive/src/attribute.rs

```diff
@@ -961,27 +968,20 @@ pub mod quickcheck {
                     ))
                 }
             }
-            NestedMeta::Meta(_) => Err(syn::Error::new_spanned(
+            _ => Err(syn::Error::new_spanned(
```

Comment:
I have changed this to avoid the wildcard. The changed error have changed yes, since syn meta is now different, but the user error is still helpful in this scenario